group_study/retrospective.md

```diff
@@ -2,22 +2,59 @@
 
 # Retrospective
```

Comment:
What is it? Log of some kind of program? What you wanted to say with this diary?